### ISSUE TITLE
Steamdeck UI has extremely low FPS when above 720p

### ISSUE THREAD

**Issue by jessicamaybe** [ValveSoftware/steam-for-linux#8706]
#### Your system information

* Distribution (e.g. Ubuntu): Arch Linux
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Running steam with gamepadui has very low FPS when rendering above 720p. The low framerate is mainly on the "home" screen. This is an issue on steam deck as well.

#### Steps for reproducing this issue:

1. opt into steamdeck steam branch
2. set resolution above 720p using gamescope 
3. launch steam using gamepadui

**Comment by 20PercentRendered** [comment]
This is a web component issue, as I experience the same issue if I open steam's library page maximized.

**Comment by Bitwolfies** [comment]
I have the same issue with the new offical beta rollout of the ui, is there any fix? It doesn't seem to affect any of my friends.

**Comment by ThisNekoGuy** [comment]
The new GPU acceleration update (`1667268053`) doesn't seem to improve this at all, just a heads up; even if you try to force it with the `-cef-force-gpu` arguement

**Comment by ThisNekoGuy** [comment]
Correction:
At 1080p, (personally) this doesn't seem to be much of an issue but it persists at 4K (3840x2160)

**Comment by papamikey** [comment]
its really bad at 1440p as well. Very strange that this happens on Linux but not Windows. On Windows 10 it renders perfectly at 1440p at my max refresh rate.  Also in the beta we have right now, it still persists. Not only that, but the overlay crashes often because of this lag I'm assuming. I hope this gets resolved soon, it's been a problem for almost a year!

**Comment by trinque** [comment]
Yep, this is currently pretty unusable in Big Picture. I'm running on Linux with a 3090.

**Comment by kzdixon** [comment]
It's been working much better now in the beta updates I've been using (with GPU acceleration enabled on AMD/7900 XTX at 4k).  
It does seem to crash/become unusable very often though; Does this every time after switching virtual desktops on i3wm and seems to only get a few toggle uses out of it when trying to edit a Steam Controller config midgame.  
Scaling for the gamepad overview is also pretty broken at 4k.

**Comment by 20PercentRendered** [comment]
Or people with a 4k monitor with steam maximized with the default library view.

**Comment by pquerner** [comment]
Confirming. Still an issue on 1716584667. (from Steam > Help > About Steam)

Nvidia 3080, X11, 550 drivers (dkms) on KDE Plasma 6.0.5, 6.9.3-arch1-1.

**Comment by mhanuszh** [comment]
Still terrible as of today on both of my PCs

Steam beta 1721420033

Debian 12, Gnome 43.9, X11, Nvidia 535.183.01, Kernel 6.1.0-17-amd64
Intel Core i5-2500, GeForce GTX 1050 Ti, 16GB DDR3

Fedora 40, Gnome 46.3.1, Wayland, Intel Mesa 24.1.2 i915, Kernel 6.9.9-200.fc40.x86_64
Intel Xeon E3-1225 v5, Intel HD Graphics P530, 8GB DDR4

**Comment by cprn** [comment]
Neither hardware, nor display server matters - I have this issue on AMD Ryzen 7 7800X3D and Radeon RX 7900 XTX.

**Comment by khalid151** [comment]
For those still having issues, make sure to enable GPU accelerated rendering (in Settings > Interface) and make sure to enable the option to "Disable GPU blocklist" (in Settings > Display) and the most important step, make sure to participate in *Steam Families Beta*.

I was having issues and recent notifications on this issue made me check gamepad UI again. After doing those steps, it started to work flawlessly.

Note that I'm using an nvidia GPU with 555.58.02 drivers.
